### ISSUE TITLE
Can't install Chai on Ubuntu 18.04: found 1 low severity vulnerability

### ISSUE THREAD

**Issue by zak100** [chaijs/chai#1371]
Hi,
I am not able to install chai. I am trying following:
`$ npm install chai`

But I am getting the message:

> npm WARN zeppelin@1.0.0 No repository field.
> npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@2.1.3 (node_modules/fsevents):
> npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
> 
> + chai@4.2.0
> updated 1 package and audited 456 packages in 2.53s
> 
> 56 packages are looking for funding
>   run `npm fund` for details
> 
> found 1 low severity vulnerability
>   run `npm audit fix` to fix them, or `npm audit` for details

`$ cat package.json`
```
{
  "name": "zeppelin",
  "version": "1.0.0",
  "description": "Testing",
  "main": "index.js",
  "dependencies": {
    "mocha": "^8.2.1",
    "web3": "^1.3.1"
  },
  "devDependencies": {
    "chai": "^4.2.0"
  },
  "scripts": {
    "preinstall": "npx npm-force-resolutions"
  },
  "resolutions": {
    "minimist": "^1.2.5"
  },
  "author": "",
  "license": "ISC"
}

`$ npm audit fix --force`

> npm WARN using --force I sure hope you know what you are doing.
> npm WARN zeppelin@1.0.0 No repository field.
> 
> added 1 package from 5 contributors in 1.221s
> 
> 56 packages are looking for funding
>   run `npm fund` for details
> 
> fixed 0 of 1 vulnerability in 456 scanned packages
> 

Please hep me solve the installation problem.

Zulfi.
zulfi6000@yahoo.com

```

**Comment by keithamus** [comment]
These are just warnings. It has installed fine. You need to add `--save-dev` to save it to your `package.json`; e.g. `npm i --save-dev chai`.

I'll close this, as things are working as intended.

**Comment by zak100** [comment]
Hi,
I am still getting the same error. I tried:$ npm i --save-dev chai
npm WARN zeppelin@1.0.0 No repository field.

+ chai@4.3.0
updated 1 package and audited 456 packages in 2.324s

59 packages are looking for funding
  run `npm fund` for details

found 1 low severity vulnerability
  run `npm audit fix` to fix them, or `npm audit` for details


   ╭───────────────────────────────────────────────────────────────╮
   │                                                               │
   │      New major version of npm available! 6.14.11 → 7.5.2      │
   │   Changelog: https://github.com/npm/cli/releases/tag/v7.5.2   │
   │               Run npm install -g npm to update!               │
   │                                                               │
   ╰───────────────────────────────────────────────────────────────╯

zulfi@lc2530hz:~$ chai

Command 'chai' not found, did you mean:

  command 'chat' from deb ppp
  command 'cai' from deb emboss
  command 'ohai' from deb ohai

Try: sudo apt install <deb name>
Please guide me how I know that it has been installed?
Zulfi.


    On Thursday, February 4, 2021, 5:13:52 AM CST, Keith Cirkel <notifications@github.com> wrote:  
 
 


These are just warnings. It has installed fine. You need to add --save-dev to save it to your package.json; e.g. npm i --save-dev chai.

I'll close this, as things are working as intended.

**Comment by zak100** [comment]
Hi,
I am still getting the same problem:
:~$ npm install chai

changed 1 package, and audited 456 packages in 3s

44 packages are looking for funding
  run `npm fund` for details

21 high severity vulnerabilities

To address issues that do not require attention, run:
  npm audit fix

Some issues need review, and may require choosing
a different dependency.

Run `npm audit` for details.
zulfi@lc2530hz:~$ npm audit fix

added 7 packages, removed 1 package, changed 63 packages, and audited 462 packages in 11s

51 packages are looking for funding
  run `npm fund` for details

# npm audit report

web3  *
Insecure Credential Storage - https://npmjs.com/advisories/877
No fix available
node_modules/web3

1 low severity vulnerability

Some issues need review, and may require choosing
a different dependency.
zulfi@lc2530hz:~$ chai

Command 'chai' not found, did you mean:

  command 'chat' from deb ppp
  command 'ohai' from deb ohai
  command 'cai' from deb emboss

Try: sudo apt install <deb name>

zulfi@lc2530hz:~$ 
Please guide me how to solve this problem.
Zulfi.
